cluster/userdata-master.yaml

```diff
@@ -192,7 +191,7 @@ coreos:
           --volume=kube,kind=host,source=/etc/kubernetes,readOnly=true \
           --mount=volume=kube,target=/etc/kubernetes \
           --net=host \
-          docker://registry.opensource.zalan.do/teapot/hyperkube:v1.7.9_coreos.0 \
+          docker://gcr.io/google_containers/hyperkube:v1.8.4 \
```

Comment:
gcr?

Reply:
*work in progress*, but thanks for the reminder because I could have totally forgot to change it :)